### PR TITLE
Find and replace all _([:alpha][:alnum]*) with _mathrm{$1} ...

### PR DESCRIPTION
**Pull request by jamesray1**
... using Libre Open Office with regular expressions enabled as an option. Man it'd be really good if you could change a commit summary after committing, on Github. I passed on that feedback to support@github.com. Additionally, there are now no search results for _{([:alpha][:alnum]*)} (also with regular expressions ticked with find and replace on LOO).

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -281,15 +281,15 @@ \subsection{The Transaction} \label{ch:transaction}
 
 \begin{equation}
 L_T(T) \equiv \begin{cases}
-(T_n, T_p, T_g, T_t, T_v, T_\mathbf{i}, T_w, T_r, T_s) & \text{if} \; T_t = \varnothing\\
-(T_n, T_p, T_g, T_t, T_v, T_\mathbf{d}, T_w, T_r, T_s) & \text{otherwise}
+(T_n, T_mathrm{p}, T_g, T_t, T_v, T_\mathbf{i}, T_w, T_r, T_s) & \text{if} \; T_t = \varnothing\\
+(T_n, T_mathrm{p}, T_g, T_t, T_v, T_\mathbf{d}, T_w, T_r, T_s) & \text{otherwise}
```
Comment (pirapira) [inline Paper.tex]: Other elements of the tuple should be treated the same way.

Comment (jamesray1) [reply Paper.tex]: Hmm, I wonder why that wasn't picked up.

Comment (jamesray1) [reply Paper.tex]: Ah I know what I did wrong.

**Comment by jamesray1** [comment]
Replaced by #499.

**Pull request closed without merging**